taxi/views.py

```diff
@@ -3,7 +3,9 @@
 from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import render, redirect, get_object_or_404
```

Comment:
The `render` function is imported twice. Consider removing the redundant import to clean up the code.